### PR TITLE
docs: Fix Cloudwatch datasource IAM policy invalid JSON

### PR DESCRIPTION
**Pull request by chancez**
JSON doesn't allow for trailing commas, so fix the cloudwatch IAM policy JSON by removing the trailing comma.

### CODE REVIEW

File: docs/sources/features/datasources/cloudwatch.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ Here is a minimal policy example:
         "logs:StartQuery",
         "logs:StopQuery",
         "logs:GetQueryResults",
-        "logs:GetLogEvents",
+        "logs:GetLogEvents"
```
Comment (oddlittlebird) [inline docs/sources/features/datasources/cloudwatch.md:83]: The comma is common to the other lines, why only remove it from the one line?

Comment (waybeforenow) [reply docs/sources/features/datasources/cloudwatch.md:83]: A trailing comma at the end of the list makes it invalid JSON; it's probably a copy/paste error. Check lines 65-70 in `cloudwatch.md`.

**Comment by oddlittlebird** [comment]
LGTM, but please update this branch from master. I think the fix for the codespell test is in there. If not, it will be soon.

**Review by oddlittlebird — APPROVED** [review]
Fix is good, just need the codespell test fix for I AM.

**Comment by chancez** [comment]
Anything else I should do here?

**Review by aknuds1 — APPROVED** [review]

**Comment by aknuds1** [comment]
Thanks for the fix! Please merge with master in order to fix the CI issue.

**Comment by chancez** [comment]
@aknuds1 rebased on top of latest master.

**Comment by aknuds1** [comment]
@chancez I think you'll need to reopen the PR?

**Comment by chancez** [comment]
I guess after rebasing the change has already been applied through a different PR. #25153

**Comment by aknuds1** [comment]
@chancez Ah OK, I didn't realize there was a duplicate PR. Thanks for the effort in any case!

**Pull request closed without merging**